Rebuild constructure, scss

yufuworks opened the pull request:
# What?
完成サイトの情報を解析して作成したバージョン
自作オリジナルは別ブランチとして保存済み

# Why?
最初に作った構造、scssが行き詰まったため

yufuworks committed:
中間

haml写し完了

yufuworks committed:
CheckedScss

SCSS調整完了
※完成品を解析して別ブランチとして作り直し

yufuworks commented:
## レイアウトプレビュー
https://gyazo.com/d8b07c44304057ed98b1cb00a50bb365

i-date commented on app/assets/stylesheets/_form.scss:
```diff
@@ -0,0 +1,42 @@
+.input-box {
+  position: relative;
+  width: 100%;
+  .input-box__text {
```
&を使用して省略しましょう。

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/_form.scss:
```diff
@@ -0,0 +1,42 @@
+    top: 10px;
+    right: 10px;
+    cursor: pointer;
+    .image-label__input {
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/_chat-side.scss:
```diff
@@ -0,0 +1,42 @@
+  &__latest-message {
+    font-size: 11px;
+  }
+}
```
最下行に空行を挿入してあげましょう。

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/_form.scss:
```diff
@@ -0,0 +1,42 @@
+  background-color: $bg-color-blue;
+  color: $font-color-white;
+  cursor: pointer;
+}
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/_main-header.scss:
```diff
@@ -0,0 +1,33 @@
+  margin-top: 30px;
+  border: solid 1px $bg-color-blue;
+  color: $bg-color-blue;
+}
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,28 @@
+      margin-bottom: 10px;
+    }
+  }
+}
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/_property.scss:
```diff
@@ -1,16 +1,7 @@
+$font-color-white: #FFFFFF;
+$bg-color-blue: #38AEF0;
+$bg-color-chat-header: #253141;
+$bg-color-chat-side: #2F3E51;
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/assets/stylesheets/application.scss:
```diff
@@ -1,6 +1,8 @@
+@import "chat-side.scss";
+@import "main-header.scss";
+@import "messages.scss";
+@import "form.scss";
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/views/chats/index.html.haml:
```diff
@@ -0,0 +1,46 @@
+        = link_to "#" do
+          .group
+            %p.group__group-name group1
+            %p.group__latest-message 画像が投稿されています
```
15 - 19行目を部分テンプレートに切り出してみてはいかがでしょうか？

yufuworks replied:
テンプレート作成しました

i-date commented on app/views/chats/index.html.haml:
```diff
@@ -0,0 +1,46 @@
+        .message
+          .upper-info
+            %p.upper-info__user username2
+            %p.upper-info__date 2019/04/19(Fri) 12:29:34
```
31 - 39行目を部分テンプレートに切り出してみてはいかがでしょうか？

yufuworks replied:
テンプレート作成しました

i-date reviewed (commented):
ご提出ありがとうございます！コメントいたしました^^

yufuworks committed:
FixTemplatesAndMadeRender

・footerパーツのセレクター名修正
 =>&使用
・各ファイルの最後の空白行を追加
・index ViewでのDBからの繰り返し予定要素を部分テンプレートとして切り出し。
 =>展開用の条件式は未実装

i-date commented on app/views/chats/_group.html.haml:
```diff
@@ -0,0 +1,4 @@
+= link_to "#" do
+  .group
+    %p.group__group-name group1
+    %p.group__latest-message 画像が投稿されています
```
最下行は空行を挿入してあげましょう。

yufuworks replied:
修正しました

i-date commented on app/views/chats/_message.html.haml:
```diff
@@ -0,0 +1,10 @@
+  .upper-info
+    %p.upper-info__user username2
+    %p.upper-info__date 2019/04/19(Fri) 12:29:34
+  %p.message__text sumple chat
```
こちらも

yufuworks replied:
修正しました

i-date commented on app/views/chats/index.html.haml:
```diff
@@ -13,10 +13,7 @@
-          .group
-            %p.group__group-name group1
-            %p.group__latest-message 画像が投稿されています
+        = render :partial => "group"
```
ハッシュロケットは古い記法なので以下のようにシンボル型で書いてあげましょう。
= render partial: "group"

yufuworks replied:
修正しました

i-date commented on app/views/chats/index.html.haml:
```diff
@@ -28,19 +25,11 @@
-          .upper-info
-            %p.upper-info__user username2
-            %p.upper-info__date 2019/04/19(Fri) 12:29:34
+        = render :partial => "message"
```
こちらも

yufuworks replied:
修正しました

yufuworks committed:
FixSyntaxes

・追加レンダーファイルの空白行追加
・render文法の修正

yufuworks commented:
# 追加したテンプレートファイルの空白行追加
# render文法の修正

i-date reviewed (approved):
早速修正していただきありがとうございます！
拝見いたしましたが、特に問題ありません！LGTMとさせていただきます^^

ご参考までに。
[WIP]は外してからレビューを依頼するとよろしいかと思います。
WIP = 作業中 なので、一通り作業が完了→WIP外す→レビュー依頼を出す、という流れになります！
